.github/workflows/ci.yml

```diff
@@ -82,7 +82,19 @@ jobs:
       - name: Install dependencies
         run: tools/flutter-dependencies.sh
       - name: Run tests
-        run: (cd packages/flutter && flutter test)
+        run: (cd packages/flutter && flutter test --coverage)
+      - name: Convert code coverage
```

Comment:
I think this will break if platform specific features are introduced. Is there a reason why you only use the coverage from 1 platform?

Reply:
I thought creating the same file 3 times in useless. Additionally, it [failed](https://github.com/parse-community/Parse-SDK-Flutter/runs/6251467226?check_suite_focus=true) on windows due to cmd being a bit different to bash.

Reply:
Could you add a short comment that this needs to consider all platforms when platform specific features are introduced?

Reply:
In the ci.yaml file?

Reply:
Perfect!